.github/workflows/build.yml

```diff
@@ -0,0 +1,34 @@
+name: build
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: windows-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          submodules: true
+      - name: Set up Python 2.7
+        uses: actions/setup-python@v2
+        with:
+          python-version: 2.7
+      - name: Prepare
+        run: .\ps1_scripts\prepare.ps1 -python2 python
+      - name: Configure
+        env:
+          TESTED_LATEST_WOT_VERSION: ${{ secrets.TESTED_LATEST_WOT_VERSION }}
+        run: |
+          .\ps1_scripts\configure.ps1 `
+            -tested_latest_wot_version "$env:TESTED_LATEST_WOT_VERSION" `
+            -build_root_dir $(Join-Path $(Split-Path -Qualifier ${env:GITHUB_WORKSPACE}) 'build') `
+            -use_short_path
+      - name: Build
+        run: .\ps1_scripts\build.ps1
+      - name: Package
+        run: .\ps1_scripts\package.ps1
+      - name: Archive production artifacts
+        uses: actions/upload-artifact@v2
+        with:
+          name: test
```

Comment:
- Do not upload whole of outputs
- Upload zip which is used distribution as artifact with date, git short sha
  - [CMake doc](https://cmake.org/cmake/help/git-master/): `${version}.${date}-${short_sha}`

Reply:
- Done
- GitHub Actions can use `GITHUB_SHA` only.
- `${version}.${short_sha}` was implemented